public/app/features/canvas/runtime/scene.tsx

```diff
@@ -366,6 +395,13 @@ export class Scene {
       targets = event.selected;
       this.updateSelection({ targets });
 
+      // @TODO handle for group/frame?
+      // just for one element for now
+      if (targets.length === 1) {
+        const element = targets[0];
+        element.addEventListener('contextmenu', (ev) => this.handleMouseEvent(ev));
```

Comment:
Figma's current UX is to select an element when bringing up the context menu and hovering over an element. Our current implementation doesn't allow you to initialize a context menu until first selecting an element, which then works well. However, on deselecting an element we can still bring up the context menu at which point the actions have no functionality (as we don't have any selected elements)

This may be a little complex to fully match Figma's UX so we can add this to our backlog to improve UX, but we probably should at least address phantom context menu with actions that have no result (when no elements are selected)

https://user-images.githubusercontent.com/22381771/168713813-a60c5c7a-2723-4237-862b-34fbbe982c4e.mov